aws/resource_aws_cloudwatch_dashboard.go

```diff
@@ -0,0 +1,139 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsCloudWatchDashboard() *schema.Resource {
+	return &schema.Resource{
+		Create:        resourceAwsCloudWatchDashboardCreate,
+		Read:          resourceAwsCloudWatchDashboardRead,
+		Update:        resourceAwsCloudWatchDashboardUpdate,
+		Delete:        resourceAwsCloudWatchDashboardDelete,
+		SchemaVersion: 1,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Sure! 😄 
here is an example: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/import_aws_cloudwatch_event_rule_test.go